tekore/client/process.py

```diff
@@ -25,5 +25,5 @@ def model_list(type_: type, top_item: str):
     Unpack items inside ``top_item`` of dict into constructors.
     """
     def post_func(json: dict):
-        return ModelList(type_(**i) for i in json[top_item])
+        return ModelList(type_(**i) if i is not None else None for i in json[top_item])
```

Comment:
Added check for `None` in `model_list` which is performed in `track.tracks_audio_features`. Right now I don't see problems, but maybe there should be separate method in `process` which explicitly checks for None, since modified method silently converts to `None`, while original method would raise if value was `None`.

Reply:
It's probably fine like that. Filtering Nones out would be wrong, but this preserves them and does not lead to confusion.